content/docs/command-reference/exp/init.md

```diff
@@ -56,10 +72,13 @@ $ dvc exp init './another_script.sh $MYENVVAR'
 
 ## Options
 
-- `-i`, `--interactive` - prompts user for the command to execute and different
-  paths for tracking outputs and dependencies, unless they are provided through
-  arguments explicitly. Interactive mode allows users to set those locations
-  from default values or omit them.
+- `-i`, `--interactive` - prompts user for the `command` to execute and for the
```

Comment:
to execute -> experiment `command` or something. otherwise it's not clear what command

Reply:
Done in 3d916568.